Core/src/Geometry/CylinderVolumeBuilder.cpp

```diff
@@ -540,8 +540,10 @@ Acts::VolumeConfig Acts::CylinderVolumeBuilder::analyzeContent(
         double rMaxD = dBounds->rMax();
         double zMinD = center.z() - 0.5 * thickness;
         double zMaxD = center.z() + 0.5 * thickness;
-        lConfig.rMin =
-            std::min(lConfig.rMin, rMinD - m_cfg.layerEnvelopeR.first);
+        lConfig.rMin = std::min((lConfig.rMin < 0 ? 0.0 : lConfig.rMin),
+                                (rMinD - m_cfg.layerEnvelopeR.first
+                                     ? 0.0
+                                     : rMinD - m_cfg.layerEnvelopeR.first));
```

Comment:
How about 

```cpp
lConfig.rMin = std::min(lConfig.rMin, rMinD - m_cfg.layerEnvelopeR.first);
lConfig.rMin = std::max(0, lConfig.rMin);
```

That does the same thing, but includes a bit fewer ternaries.

Reply:
Looks good to me.